scripts/distribution/windows/build-all.ps1

```diff
@@ -5,19 +5,16 @@ Write-Output "cargo version: $(cargo --version)"
 Write-Output "flatc version: $(flatc --version)"
 Write-Output "protoc version: $(protoc --version)"
 
-# Set the default rust toolchain so that consensus rust dependencies use it.
-rustup default $rustVersion-x86_64-pc-windows-gnu
-
 Write-Output "Building consensus..."
 stack build
 if ($LASTEXITCODE -ne 0) { throw "Failed building consensus" }
 
 Write-Output "Building node..."
-stack exec -- cargo build --manifest-path concordium-node\Cargo.toml --release
+stack exec -- cargo build --manifest-path concordium-node\Cargo.toml --release --locked
 if ($LASTEXITCODE -ne 0) { throw "Failed building node" }
 
 Write-Output "Building the collector..."
-cargo +$rustVersion-x86_64-pc-windows-msvc build --manifest-path collector\Cargo.toml --release
```

Comment:
This seems to be wrong according to README

Reply:
Seems like this is to force the toolchain according to the cargo [documentation](https://doc.rust-lang.org/cargo/commands/cargo.html#common-options) I think it's best to keep it.

Reply:
This is to force a different toolchain. According to docs:

`  - For building the node runner service (optional), the toolchain `1.73-x86_64-pc-windows-msvc`  is required, which can be installed with the command: `rustup toolchain install 1.73-x86_64-pc-windows-msvc`.
`

This is not any such service. I want to use the default service here.